configs/prod/bundles.yml

```diff
@@ -345,20 +345,50 @@ objects:
       - name: acs
         skus:
           - MW02162
+          - MW02162RN
+          - MW02162S
           - MW02162F3
+          - MW02162F3RN
+
           - MW02159
+          - MW02159RN
+          - MW02159S
+          - MW02159F3
+          - MW02159F3RN
+
           - MW02164HR
           - MW02161HR
           - MW02163MO
           - MW02160MO
+
           - MW02165
-          - MW02166
+          - MW02165RN
+          - MW02165S
           - MW02165MO
+          - MW02165F3
+          - MW02165F3RN
+
+          - MW02166
+          - MW02166RN
+          - MW02166S
           - MW02166MO
+          - MW02166F3
+          - MW02166F3RN
+
```

Comment:
Would you also mind removing the newlines to match the formatting of the other SKUs? 

It might mess up the reading of the file into the configmap! 

Thank you so much!